### ISSUE TITLE
weights in cross entropy function

### ISSUE THREAD

**Issue by rst1991** [alireza-nasiri/SoundCLR#5]
def cross_entropy_one_hot(input, target):
	_, labels = target.max(dim=1)
	return nn.CrossEntropyLoss(weight=class_weights)(input, labels).

What is weight=class_weights for different datasets

**Comment by alireza-nasiri** [comment]
Here, it was unnecessary so I removed it but in general weights can be in the case of imbalanced datasets where there is an unequal distribution of classes within dataset.
